ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import time
 
-from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.page_objects.page_navigator import PageNavigator
```

Comment:
Why did you move PageNavigator class from base_ui file?

Reply:
This is a module of the page and it has navigation methods, same as any DataFoundationTab page or object, no difference. 
It does not have basic ui functions such as do_click(), get_element_attribute(), take_screenshot(), etc related to base_ui functionality. 
Plus it seriously reduce complexity of the base_ui.py module under `ocs_ci/ocs/ui/page_objects` directory. 
In other words every "page object" including PageNavigator now has it's own module, functionality may continue grow and we always will have full order.